### ISSUE TITLE
About the results on the CNN dataset (such as mir, nus and coco) in the paper.

### ISSUE THREAD

**Issue by Cherishnoobs** [XinyuXia97/UCMFH#6]
Can you provide the hyperparameter Settings for the CNN dataset, such as the learning rates of the three optimizers, and the weights for loss_1 and loss_2? The dataset used at present is provided by another work "CIRH" of Teacher Zhu Lei, but in the case of only adjusting the learning rate, the experiment is up to 4 percentage points away from the effect described in the paper. Can you provide more detailed experimental parameters in order to quote your work with us?

**Comment by XinyuXia97** [comment]
Thank you very much for your attention to our work. The code and checkpoints can be obtained from the following link: [https://drive.google.com/drive/folders/1RqgX4gYE19Uh3KXzYl4NHq0UWQ9JhtKs?usp=sharing](url)

**Comment by Cherishnoobs** [comment]
Thank you very much！

**Comment by Cherishnoobs** [comment]
Thank you for providing the link, please apply through my visit.

**Comment by XinyuXia97** [comment]
> Thank you for providing the link, please apply through my visit.

The link is now accessible to you.
